server/schema/mutations.js

```diff
@@ -185,6 +191,23 @@ const mutation = new GraphQLObjectType({
       resolve(parentValue, {id}) {
         return Post.remove({_id: id});
       }
+    },
+    followUser: {
+      type: UserType,
+      args: {
+        userId: {type: new GraphQLNonNull(GraphQLID)}
+      },
+      async resolve(parentValue, {userId}, {user}) {
+        if (!user) {
```

Comment:
how come we are not using context any more? @nashawn-griffith

Reply:
Same reason as above

Reply:
@nashawn-griffith where did you destructure it?

Reply:
{user} - this is where it was destructured

Reply:
ohok, I see what going on.